app/templates/teachers/share-licenses-component.jade

```diff
@@ -0,0 +1,43 @@
+#share-licenses-component.modal-body 
+  .text-center.subheader-text
+    p.small
+      {{ $t("share_licenses.subheader") }}
+  
+  h4.m-t-3.input-label
+    {{ $t("share_licenses.add_teacher_label") }}
+  
+  form.row.form-group
+    .col-xs-8
+      input.form-control(v-model:value.trim="teacherSearchInput")
+    .col-xs-4
+      button.add-teacher-btn.btn.btn-navy(v-on:click.prevent="addTeacher")
```

Comment:
I believe `submit` and `click` (on a button) are completely equivalent in a form context, with Vue. Preventing submit on clicking the button also prevents submit on enter.